docs/django-migrations-standards.md

```diff
@@ -0,0 +1,29 @@
+# Django Migrations Standards
+
+From Django 3.2 documentation on [Migrations](https://docs.djangoproject.com/en/3.2/topics/migrations/#module-django.db.migrations)
+
+> Migrations are Django’s way of propagating changes you make to your models (adding a field, deleting a model, etc.) into your database schema. They’re designed to be mostly automatic, but you’ll need to know when to make migrations, when to run them, and the common problems you might run into.
+
+For creating migrations use Django's built-in commands [here](https://docs.djangoproject.com/en/3.2/topics/migrations/#the-commands).
+
+### Adding New Model Field and Migrations
```

Comment:
Nit: "and Migrations" makes the plural agreement weird. Dropping it flows better for me.

"Adding a New Model Field"
or
"Adding New Model Fields"

Reply:
[8242ef8](https://github.com/mozilla/fx-private-relay/pull/3930/commits/8242ef8bd6865cdaf1f6483f17bcc09d403daf1c)